{
	"document_title": "Security update for python-Pillow",
	"document_type": "SUSE Patch",
	"document_publisher": {
		"type": "Vendor",
		"contact_details": "security@suse.de",
		"issuing_authority": "SUSE Security Team"
	},
	"document_tracking": {
		"identification": {
			"id": "SUSE-SU-2024:0185-1"
		},
		"status": "Final",
		"version": "1",
		"revision_history": {
			"revision": {
				"number": "1",
				"date": "2024-01-23T12:33:25Z",
				"description": "current"
			}
		},
		"initial_release_date": "2024-01-23T12:33:25Z",
		"current_release_date": "2024-01-23T12:33:25Z",
		"generator": {
			"engine": "cve-database/bin/generate-cvrf.pl",
			"date": "2017-02-24T01:00:00Z"
		}
	},
	"document_notes": [
		{
			"text": "Security update for python-Pillow",
			"title": "Topic",
			"type": "Summary"
		},
		{
			"text": "This update for python-Pillow fixes the following issues:\n\n- CVE-2023-50447: Fixed arbitrary code execution via the environment parameter. (bsc#1219048)\n",
			"title": "Details",
			"type": "General"
		},
		{
			"text": "The CVRF data is provided by SUSE under the Creative Commons License 4.0 with Attribution (CC-BY-4.0).",
			"title": "Terms of Use",
			"type": "Legal Disclaimer"
		},
		{
			"text": "SUSE-2024-185,openSUSE-SLE-15.5-2024-185",
			"title": "Patchnames",
			"type": "Details"
		}
	],
	"documentdistribution": "Copyright SUSE LLC under the Creative Commons License 4.0 with Attribution (CC-BY-4.0)",
	"document_references": [
		{
			"type": "Self",
			"url": "https://www.suse.com/support/update/announcement/2024/suse-su-20240185-1/",
			"description": "Link for SUSE-SU-2024:0185-1"
		},
		{
			"type": "Self",
			"url": "https://lists.suse.com/pipermail/sle-security-updates/2024-January/017746.html",
			"description": "E-Mail link for SUSE-SU-2024:0185-1"
		},
		{
			"type": "Self",
			"url": "https://www.suse.com/support/security/rating/",
			"description": "SUSE Security Ratings"
		},
		{
			"type": "Self",
			"url": "https://bugzilla.suse.com/1219048",
			"description": "SUSE Bug 1219048"
		},
		{
			"type": "Self",
			"url": "https://www.suse.com/security/cve/CVE-2023-50447/",
			"description": "SUSE CVE CVE-2023-50447 page"
		}
	],
	"product_tree": {
		"branch": [
			{
				"type": "Product Family",
				"name": "openSUSE Leap 15.5",
				"branch": [
					{
						"type": "Product Name",
						"name": "openSUSE Leap 15.5",
						"full_product_name": {
							"text": "openSUSE Leap 15.5",
							"product_id": "openSUSE Leap 15.5",
							"cpe": "cpe:/o:opensuse:leap:15.5"
						}
					}
				],
				"full_product_name": {}
			},
			{
				"type": "Product Version",
				"name": "python3-Pillow-7.2.0-150300.3.6.1",
				"full_product_name": {
					"text": "python3-Pillow-7.2.0-150300.3.6.1",
					"product_id": "python3-Pillow-7.2.0-150300.3.6.1"
				}
			},
			{
				"type": "Product Version",
				"name": "python3-Pillow-tk-7.2.0-150300.3.6.1",
				"full_product_name": {
					"text": "python3-Pillow-tk-7.2.0-150300.3.6.1",
					"product_id": "python3-Pillow-tk-7.2.0-150300.3.6.1"
				}
			}
		],
		"relationship": [
			{
				"product_reference": "python3-Pillow-7.2.0-150300.3.6.1",
				"relation_type": "Default Component Of",
				"relates_to_product_reference": "openSUSE Leap 15.5"
			},
			{
				"product_reference": "python3-Pillow-tk-7.2.0-150300.3.6.1",
				"relation_type": "Default Component Of",
				"relates_to_product_reference": "openSUSE Leap 15.5"
			}
		]
	},
	"vulnerability": [
		{
			"notes": [
				{
					"text": "Pillow through 10.1.0 allows PIL.ImageMath.eval Arbitrary Code Execution via the environment parameter, a different vulnerability than CVE-2022-22817 (which was about the expression parameter).",
					"title": "Vulnerability Description",
					"type": "General"
				}
			],
			"cve": "CVE-2023-50447",
			"product_statuses": [
				{
					"type": "Fixed",
					"product_id": [
						"openSUSE Leap 15.5:python3-Pillow-7.2.0-150300.3.6.1",
						"openSUSE Leap 15.5:python3-Pillow-tk-7.2.0-150300.3.6.1"
					]
				}
			],
			"threats": [
				{
					"type": "Impact",
					"description": "important"
				}
			],
			"cvss_score_sets": {
				"score_set_v_2": {},
				"score_set_v_3": {
					"base_score_v_3": "8.4",
					"vector_v_3": "CVSS:3.1/AV:L/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H"
				},
				"scoreset": {}
			},
			"remediations": [
				{
					"type": "Vendor Fix",
					"description": {
						"text": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
						"lang": "en"
					},
					"url": "https://www.suse.com/support/update/announcement/2024/suse-su-20240185-1/"
				}
			],
			"references": [
				{
					"url": "https://www.suse.com/security/cve/CVE-2023-50447.html",
					"description": "CVE-2023-50447"
				},
				{
					"url": "https://bugzilla.suse.com/1219048",
					"description": "SUSE Bug 1219048"
				}
			]
		}
	]
}
